packages/office-ui-fabric-react/src/components/__snapshots__/Keytips.Button.Example.tsx.shot

```diff
@@ -124,7 +124,7 @@ exports[`Component Examples renders Keytips.Button.Example.tsx correctly 1`] = `
     </div>
   </button>
   <button
-    aria-describedby="id__4 ktp-layer-id ktp-1-b"
+    aria-describedby="id__4  ktp-layer-id ktp-1-b"
```

Comment:
We're sure this won't cause any issues, correct? I was wondering if each arg should be trimmed too.

Reply:
> We're sure this won't cause any issues, correct? I was wondering if each arg should be trimmed too.

Correct, see https://codepen.io/kevintcoughlin/pen/oVNoPj

```shell
s.codepen.io, Is read by narrator, button,
Description One Description Two, 
```

Though I'm willing to trim each arg if you think its worthwhile re: runtime cost.

Reply:
FWIW I think fixing the bug is more ROI than optimizing the white-space character here since screen readers will work in the above case.

Reply:
I don't want to do it without good reason.. if HTML attributes in general are tolerant of extra whitespaces across all browsers that's good enough for me.